modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -28,35 +27,41 @@
     private static final int ELASTICSEARCH_DEFAULT_TCP_PORT = 9300;
 
     /**
-     * Elasticsearch Docker base URL
+     * Elasticsearch Docker base image
      */
-    private static final String ELASTICSEARCH_DEFAULT_IMAGE = "docker.elastic.co/elasticsearch/elasticsearch";
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch");
```

Comment:
I'm wondering if we could make this static constant public so people can simply do something like:

```java
new ElasticsearchContainer(ElasticsearchContainer.DEFAULT_IMAGE_NAME.withVersion("7.9.2"));
```